gp-includes/things/project.php

```diff
@@ -232,20 +232,22 @@ public function normalize_fields( $args ) {
 	 */
 	public function update_path() {
 		global $wpdb;
-		$old_path = isset( $this->path ) ? $this->path : '';
+		$old_path       = isset( $this->path ) ? $this->path : '';
 		$parent_project = $this->get( $this->parent_project_id );
-		if ( $parent_project )
-			$path = gp_url_join( $parent_project->path, $this->slug );
-		elseif ( ! $wpdb->last_error )
-			$path = $this->slug;
-		else
+		if ( $parent_project ) {
+			$path          = gp_url_join( $parent_project->path, $this->slug );
+		} elseif ( ! $wpdb->last_error ) {
+			$path          = $this->slug;
+		} else {
```

Comment:
The alignment is incorrect as the lines are not after each other.

Reply:
Indeed, I guess it was formated before the line break.
Fixed.